src/stdatamodels/jwst/datamodels/wcs_ref_models.py

```diff
@@ -458,6 +459,88 @@ def to_fits(self):
         raise NotImplementedError("FITS format is not supported for this file.")
 
 
+class MiriWFSSSpecwcsModel(ReferenceFileModel):
+    """A model for a reference file of type "specwcs" for MIRI WFSS."""
+
+    schema_url = "http://stsci.edu/schemas/jwst_datamodel/specwcs_miri_wfss.schema"
+    reftype = "specwcs"
+
+    def __init__(
+        self,
+        init=None,
+        displ=None,
+        dispx=None,
+        dispy=None,
+        invdispl=None,
+        orders=None,
+        fwcpos_ref=None,
+        **kwargs,
+    ):
+        """
+        Initialize the model.
+
+        Parameters
+        ----------
+        init : str, None, optional
+            File name for input file in ASDF format from which to initialize the model.
+        displ : `~astropy.modeling.Model`
+            MIRI WFSS wavelength dispersion model
+        dispx : `~astropy.modeling.Model`
+            MIRI WFSS row dispersion model
+        dispy : `~astropy.modeling.Model`
+            MIRI WFSS column dispersion model
+        invdispl : `~astropy.modeling.Model`
+            MIRI WFSS inverse wavelength dispersion model
+        orders : `~astropy.modeling.Model`
+            MIRI WFSS orders, matched to the array locations of the
+            dispersion models
+        fwcpos_ref : float
+            The reference value for the filter wheel position
```

Comment:
I don't see this in the schema or in https://jwst-crds-test.stsci.edu/browse/jwst_miri_specwcs_0137.asdf
Should it be in those or removed from here? Thanks for helping me understand this.